test/functional/test_framework/test_node.py

```diff
@@ -212,6 +212,7 @@ def wait_for_rpc_connection(self):
         poll_per_s = 4
         for _ in range(poll_per_s * self.rpc_timeout):
             if self.process.poll() is not None:
+                print('process poll is not NONE', self.process.poll()) #this outputs me 1
```

Comment:
On startup failures or general test failures, you can combine the logs to see the issue:

```
$ ./test/functional/combine_logs.py -c /tmp/bitcoin_func_test_wnpgbmh_  | tail 
                                     File "/home/marco/workspace/btc_bitcoin_core/test/functional/test_framework/test_framework.py", line 419, in start_node
                                       node.wait_for_rpc_connection()
                                     File "/home/marco/workspace/btc_bitcoin_core/test/functional/test_framework/test_node.py", line 217, in wait_for_rpc_connection
                                       'bitcoind exited with status {} during initialization'.format(self.process.returncode)))
                                   test_framework.test_node.FailedToStartError: [node 0] bitcoind exited with status 1 during initialization
 test  2020-04-08T00:11:49.629000Z TestFramework (DEBUG): Closing down network thread 
 test  2020-04-08T00:11:49.680000Z TestFramework (INFO): Stopping nodes 
 test  2020-04-08T00:11:49.681000Z TestFramework.node0 (DEBUG): Stopping node 

 node0 stderr Error: Command line contains unexpected token 'persistmempool=0', see bitcoind -h for a list of options.

Reply:
See also https://github.com/bitcoin/bitcoin/tree/master/test#test-logging

I just ran the test with
```
$ ./test/functional/p2p_blocksonly.py --tracerpc -l DEBUG